lib/gemini.js

```diff
@@ -28,123 +29,125 @@ function requireWithNoCache(moduleName) {
     return result;
 }
 
-module.exports = inherit(ChainedEmitter, {
+module.exports = inherit(EventEmitter, {
```

Comment:
Why?
`ChainedEmmiter` was there for the plugins needs. Basically, it allows to return a promise from event handler and delay the further execution until that promise resolves. This allows, for example, to launch `SauceConnect` or ssh-tunnel before any tests are launched.